icons/shave-face.svg

```diff
@@ -0,0 +1,19 @@
+<svg
```

Comment:
Hard to identify IMO.

Reply:
![image](https://github.com/lucide-icons/lucide/assets/7797479/c3ed1da0-20c6-42e3-8276-4a20f2b8f575)